app/controllers/comments_controller.rb

```diff
@@ -7,6 +7,11 @@ def create
     @comment.article_id = params[:article_id]
     @comment.user_id = current_user.id
     if @comment.save
+
+      article = Article.find(@article)
+      user = User.find(article.user_id)
+      UserMailer.comment_create(user).deliver
```

Comment:
fat model skinny controller.

this should go in a after_create in the Comment model